lib/ansible/modules/system/systemd.py

```diff
@@ -307,7 +307,7 @@ def main():
     # check service data, cannot error out on rc as it changes across versions, assume not found
     (rc, out, err) = module.run_command("%s show '%s'" % (systemctl, unit))
 
-    if out.find('ignoring request') != -1:
+    if err.find('ignoring request') != -1:
```

Comment:
Could you explain the need to check stdout? `systemctl show <service>` dumps a lot of information about the service being changed. It might be unusual, but the`Description` field for example could contain the text 'ignoring request'.